.github/quarkus-github-lottery.yml

```diff
@@ -0,0 +1,38 @@
+notifications:
+  createIssues:
+    repository: "quarkusio/quarkus-github-lottery-reports"
+buckets:
+  triage:
+    # TODO: No such label on quarkusio... ? This won't work without a label.
+    label: "triage/needs-triage"
```

Comment:
See:

* https://github.com/quarkusio/quarkus-github-lottery?tab=readme-ov-file#triage

Reply:
created!

Reply:
Thanks. We also need this label to be applied automatically to new issues... On quarkusio/quarkus I think that's the bot's responsibility.

Here I suppose we could add the label as part of issue templates? Or does it make sense to enable the bot here @gsmet ?

Reply:
I dunno. Personally I'm not sure I will have the time to triage things here and we don't have that many areas.

Looks a bit too much to my taste.

But I'm not the one who asked for this so feel free to ignore me :).